### ISSUE TITLE
request to https server fails with bad handshake

### ISSUE THREAD

**Issue by jacob-keller** [requests/requests#4619]
Summary.

I am attempting to connect to an https server (unfortunately private internal server) which request.get() fails to connect, but curl succeeds. Specifically, I'm using gertty and attempting to connect to an internal gerrit server.

## Expected Result

python requests library succeeds to connect and allows me to work with the server.

## Actual Result

the connection fails with:

requests.exceptions.SSLError: HTTPSConnectionPool(host='<gerrit server>', port=443): Max retries exceeded with url: /gerrit/a/config/server/version (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'ssl3_read_bytes', 'sslv3 alert handshake failure')],)",),))

## Reproduction Steps

```python
import requests

requests.get("gerrit-server")
```

## System Information

    $ python -m requests.help

```
{
  "chardet": {
    "version": "3.0.4"
  },
  "cryptography": {
    "version": "2.0.2"
  },
  "idna": {
    "version": ""
  },
  "implementation": {
    "name": "CPython",
    "version": "2.7.14"
  },
  "platform": {
    "release": "4.15.15-300.fc27.x86_64",
    "system": "Linux"
  },
  "pyOpenSSL": {
    "openssl_version": "1010006f",
    "version": "17.2.0"
  },
  "requests": {
    "version": "2.18.4"
  },
  "system_ssl": {
    "version": "1010008f"
  },
  "urllib3": {
    "version": "1.22"
  },
  "using_pyopenssl": true
}
```

At first, I thought this was because the certificate was invalid, but I tried using curl, and it succeeded (well, ok it failed with unauthorized, since I haven't provided login credentials yet, but it did succeed the TLS handshake).

I used wireshark to compare the client hello message with the one from curl, and I found that the only major difference was that the successful curl request included more cipher suites, and the server appears to want to select one of those, so when the requests base client hello message is sent, it gets rejected.

Specifically, the server wants to use TLS_RSA_WITH_3DES_EDE_CBC_SHA, which is supported by curl, but not by the requests connection.

Any idea what is going on here, or whether there's a way to work around this issue in gertty? I am unable to use gertty with my internal server unless this is fixed.

**Comment by jacob-keller** [comment]
I *believe* this issue is very similar to an issue in scrapy: https://github.com/scrapy/scrapy/issues/2717 as that was what led me down the path of checking the cipher suite to see if it was sending the same values.

**Comment by nateprewitt** [comment]
Hi @jacob-keller, Requests no longer supports 3DES out of the box because of the known security vulnerabilities with it. There are several issues in the closed section (most notable #3774) describing how to work around this if you must. There's also a [blog post](https://lukasa.co.uk/2017/02/Configuring_TLS_With_Requests/) one of the maintainers wrote with more detailed information.

I would suggest talking with whomever is in charge of your infrastructure about updating the server, otherwise, the above solution will work.
